### ISSUE TITLE
Git not getting views

### ISSUE THREAD

**Issue by jimgwhit** [nova-framework/framework#707]
If I install nova using composer, all is fine, but for some reason github (git) isn't getting the Welcome view folder, It stays empty on my master.  Is that normal, or  should it be getting that as well. Whats funny is it's in the fork  
https://github.com/jimgwhit/framework/tree/master/app/Views/Welcome  
just doesn't pull into master (local) branch somehow.  I am going to try again.  
Tryed again and  
A git fetch upstream produces
~~~
C:\Users\jimcompq\Documents\GitHub\framework>git checkout master
D       app/Views/Welcome/SubPage.php
D       app/Views/Welcome/Welcome.php
~~~
They never show in master.  
As a work around I copied and pasted them in from a recent install, probably not right, but I had no problen when running git commands again:  
* git remote -v
* git fetch upstream
* git checkout master
* git merge upstream/master
* git push origin master  
I normally run these.  I just don't see how they were in my fork, but not master.

**Comment by LuckyCyborg** [comment]
@jimgwhit Jim, you are not using right the GIT.

First of all, you should do not use **checkout** in a your Composer installed application.

To create the tree, use:

```
composer create-project nova-framework/framework novaframework -s dev
```

Then, for updates, from the site root, just do:

```
git pull
```

To NOTE that Composer installation is NOT supposed to be used with a fork. It is about using in a local development copy.

For another GIT cookies, please read its documentation, it is no fault into Nova (v3) and its repositories, then I suppose that this Issue is **Invalid**.

**Comment by jimgwhit** [comment]
@LuckyCyborg they are two different things. I have a git folder on my computer where I maintain an up to date framework.  You know, you have
* master is on local
* origin is the forked copy
* upstream Dave's main framework 
Standard git stuff.  I was only saying if I do a composer install (nothing git related here) under htdocs, all of the framework gets installed, with no problems.  
However in the other git folder, with nothing to do with composer, if I use the git commands to update the master, all gets updated except the Welcome view folder.  Somehow it never gets added / updated in master.  I am just asking is it supposed to be there or not, it is weird that it was missing.  Yesterday when I did these
* git remote -v
* git fetch upstream
* git checkout master
* git merge upstream/master
* git push origin master  
Even your latest PR #701 was properly updated in master.  But the Welcome view folder was still missing.  That's when I decided to just copy it in there. Just wondering if it's something wrong.
